lib/accessors/email-string.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const asString = require('./string')
+
+// eslint-disable-next-line no-control-regex
```

Comment:
Control characters (ex. `\u0001`) in regex are considered as invalid by Eslint, but in this case I intentionally using them to comply with the RFC, so I disabled the rule for this line.